From 449a1a54352000bdcfd99f779930ff956a510469 Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Tue, 11 Sep 2018 17:12:25 +0200
Subject: [PATCH 243/249] wiki: Remove custom TWRP link from pioneer

* No longer necessary as official TWRP does no longer
  verify public key of payload.bin

Change-Id: Iccd4869477cd72421b5caf46d375fb79f78f26a9
---
 _data/devices/pioneer.yml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/_data/devices/pioneer.yml b/_data/devices/pioneer.yml
index a9e03de..dc1454d 100644
--- a/_data/devices/pioneer.yml
+++ b/_data/devices/pioneer.yml
@@ -11,7 +11,6 @@ cpu: Cortex-A53
 cpu_cores: '8'
 cpu_freq: 2.2 GHz
 current_branch: 15.1
-custom_twrp_link: https://www.androidfilehost.com/?fid=3700668719832241508
 depth: 9.7 mm (0.38 in)
 download_boot: With the device powered off, hold <kbd>Volume Up</kbd> and connect the USB cable. The notification light should turn blue.
 gpu: Adreno 508
-- 
2.17.1

